python/nav/ipdevpoll/plugins/juniperalarm.py

```diff
@@ -0,0 +1,84 @@
+#
+# Copyright (C) 2022 Sikt
+#
+# This file is part of Network Administration Visualized (NAV).
+#
+# NAV is free software: you can redistribute it and/or modify it under
+# the terms of the GNU General Public License version 3 as published by
+# the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+# FITNESS FOR A PARTICULAR PURPOSE. See the GNU General Public License for
+# more details.  You should have received a copy of the GNU General Public
+# License along with NAV. If not, see <http://www.gnu.org/licenses/>.
+#
+"""Juniper chassis/system alarms"""
+
+from twisted.internet import defer
+from twisted.internet.defer import returnValue
+
+from django.db import transaction
+
+from nav.event2 import EventFactory
+from nav.mibs.juniper_alarm_mib import JuniperAlarmMib
+from nav.models import event
+from nav.ipdevpoll import Plugin, db
+
+YELLOW_EVENT = EventFactory(
+    'ipdevpoll', 'eventEngine', 'juniperYellowAlarmState', 'juniperYellowAlarmNonZero', 'juniperYellowAlarmZero'
```

Comment:
Might want to discuss the alert names with the CNaaS team. As you know, one of the hardest problems in programming is naming things ;)

Reply:
Let's see how long it takes to get some feedback :)

Reply:
I popped them a chat question, let's see if they have any preference at all...

Reply:
I popped them a chat question, let's see if they have any preference at all...